.gitignore

```diff
@@ -9,3 +9,5 @@ pretrained_checkpoints
 *.egg-info/
 build/
 dist/
+
+.venv/
```

Comment:
```suggestion
```

This is up to each developper. You need to use a global gitignore (otherwise we end up with an infinite list of every possible setup)